src/components/Calculator.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+// import { Link } from 'react-router-dom';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.